src/Tools/ExternalAccess/LiveShare/Rename/RoslynRenameService.Exports.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.LiveShare.Rename
+{
+    [ExportLanguageService(typeof(IEditorInlineRenameService), StringConstants.CSharpLspLanguageName), Shared]
+    internal class CSharpLspRenameService : RoslynRenameService
```

Comment:
hmm.. wondering what we should do this for LSP. probably work item needed?

Reply:
https://github.com/dotnet/roslyn/projects/45#card-20619668